contrib/cloudflare.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Cloudflare DNS Updater
+Documentation=https://github.com/favonia/cloudflare-ddns/blob/main/README.markdown
+Wants=network-online.target
+
+[Install]
+WantedBy=multi-user.target
+
+[Service]
+Type=simple
+EnvironmentFile=/etc/cloudflare-ddns/cloudflare.service.env
+Delegate=yes
+LimitNOFILE=infinity
+LimitNPROC=infinity
+LimitCORE=infinity
+TimeoutStartSec=0
```

Comment:
```suggestion
```
It seems to me that (1) with `Type=simple`, this should not be needed, and (2) the documentation seems to indicate `TimeoutStartSec=infinity` is the correct way to disable the timeout logic. Could you possibly help me confirm it?